src/python/ensembl/io/genomio/database/factory.py

```diff
@@ -87,13 +87,14 @@ def format_db_data(server_url: URL, dbs: List[str], brc_mode: bool = False) -> L
 
 def get_core_dbs_metadata(
     server_url: URL,
+    *,
```

Comment:
for my understanding, why do we need to specify this?

Reply:
This forces all the arguments after the `*` to be keyworded, e.g. if you were to do `get_core_dbs_metadata("mysql://user@localhost:42", "my_prefix")`, the code will raise an error as the second parameter needs to have a keyword associated to it. `get_core_dbs_metadata("mysql://user@localhost:42", prefix="my_prefix")` would be the correct code.